spam/01_spam_tutorial.py

```diff
@@ -566,9 +619,12 @@ def short_comment(x):
 
 # %%
 # Download the spaCy english model
-# If you see an error in the next cell, restart the kernel
 # ! python -m spacy download en_core_web_sm
 
+# %% [markdown]
```

Comment:
When might this happen? It seems better to do this check in the very first cell (where we do other housekeeping).